### PR TITLE
Add node-tracing-example, node-serverless-app

### PR DESCRIPTION
**Pull request by christineyen**
We definitely have example Node apps, just not yet in the examples repo. Pulls from https://github.com/honeycombio/tracing-workshop-node and as-yet-unreleased https://github.com/honeycombio/instrumentation-tutorial work with an attempt at making the READMEs consistent with other `examples/*/README.md`s.

### CODE REVIEW

File: node-serverless-app/README.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+It contains examples of:
+
+- Baseline Beeline usage (not in an Express app)
+- Capture of custom metadata on Beeline-generated events
+- Definition of custom spans to augment traces
+- Continuing a propagated trace (if applicable), e.g. if the client initiated a trace via another Honeycomb Beeline
+- Error handling with a trace
+
+We've intentionally included the `node_modules` directory in hopes of simplifying dependencies.
```
Comment (eanakashima) [inline node-serverless-app/README.md]: Err, this part isn't true in this repo, right?

File: node-serverless-app/README.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+## Usage:
+
+Find your Honeycomb write key at https://ui.honeycomb.io/account, then make it available to your Lambda function.
```
Comment (eanakashima) [inline node-serverless-app/README.md]: s/write key/api key/?

File: node-tracing-example/README.md [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+
+It contains examples of:
+
+- Baseline Beeline usage in an Express app
+- Capture of custom metadata on Beeline-generated events
+- Definition of custom spans to augment traces
+- In-app tracing (across functions within the same service)
+- Distributed tracing (across services)
+
+It requires Node 8+ and has intentionally included the `node_modules` directory in hopes of simplifying dependencies.
```
Comment (eanakashima) [inline node-tracing-example/README.md]: Same question about node modules dir

File: node-tracing-example/README.md [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+
+## Usage:
+
+Find your Honeycomb write key at https://ui.honeycomb.io/account, then:
```
Comment (eanakashima) [inline node-tracing-example/README.md]: Same question about write key vs. api key

File: node-serverless-app/README.md (hunk 1 above)
Comment (christineyen) [reply node-serverless-app/README.md]: ha! thanks. it was true until i realized how large those directories would be, thanks for the catch(es)!

**Comment by christineyen** [comment]
:ludicrousspeed:

**Pull request merged**